split.js

```diff
@@ -143,6 +156,7 @@ Split = function (ids, options) {
             pair.gutter = gutter;
         }
 
-        el.style.width = 'calc(' + options.widths[i] + '% - ' + options.gutterWidth / 2 + 'px)';
```

Comment:
Curious about the `toFixed(0)`, it makes the dragging a little choppy. Do you need an integer for the new calculations?